app/views/shared/check_answers/_assets.html.erb

```diff
@@ -13,11 +13,15 @@
 
 <section class="print-no-break">
   <%= render "shared/check_answers/vehicles", read_only:, individual: %>
+</section>
 
+<section class="print-no-break">
```

Comment:
This only goes around a heading but these should go around both a heading and a summary card so they "stick together"